pyfar/dsp/filter/fractional_octaves.py

```diff
@@ -50,7 +50,7 @@ def fractional_octave_frequencies(
         raise ValueError(
             "The second frequency needs to be higher than the first.")
 
-    if num_fractions in [1, 3]:
+    if num_fractions in [1, 3] and (f_lims[0] > 15) and (f_lims[1] < 22e3):
```

Comment:
Why not
```suggestion
    if num_fractions in [1, 3] and (f_lims[0] >= 25) and (f_lims[1] <= 20e3):
```
as in the Standard?

Reply:
changed it to exact limits, could not use ``25*2**(-1/6)`` because its larger than 20 and 20 is the standart input of frequency_range